docs/src/api/class-response.md

```diff
@@ -48,6 +48,27 @@ Contains a boolean stating whether the response was successful (status in the ra
 
 Returns the matching [Request] object.
 
+## async method: Response.securityDetails
+- returns: <[null]|[Object]>
+  - `issuer?` <[string]> Common Name component of the Issuer field
```

Comment:
`issuer` <[string]> Common Name component of the Issuer field. Optional.

Reply:
This 'Optional' in the end will make it optional. I'm really sorry about this one.